packages/warehouses/src/ssh/sshTunnel.ts

```diff
@@ -275,9 +275,7 @@ export class SshTunnel<T extends CreateWarehouseCredentials> {
                             `Failed to connect to remote host: ${this.originalCredentials.host}:${this.originalCredentials.port}`,
                         );
 
-                        throw new WarehouseConnectionError(
-                            `Could not open SSH tunnel: ${getErrorMessage(e)}`,
-                        );
+                        throw new SshTunnelError(getErrorMessage(e));
```

Comment:
could we keep the previous message? 
like

```suggestion
                        throw new SshTunnelError(`Could not open SSH tunnel: ${getErrorMessage(e)}`);
```